res/rococo/rococo-parachain-state

```diff
@@ -1 +1 @@
-0x000000000000000000000000000000000000000000000000000000000000000000cf53fa329f52a92f921debd7e725942371da41a31047009a4feb9cc4d02d96b403170a2e7597b7b7e3d84c05391d139a62b157e78786d8c082f29dcf4c11131400
```

Comment:
Is this change needed or just a local build change?

Reply:
It is needed for this release. This is actual state registered in rococo. We might want to delete whole `res` dir after #49